web/massastation/src/pages/Search/Search.tsx

```diff
@@ -0,0 +1,115 @@
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useResource } from '../../custom/api';
+import { DomainsModel } from '../../models/DomainsModel';
+
+import {
+  Description,
+  SidePanel,
+  Input,
+  TextArea,
+  Button,
+  DragDrop,
+} from '@massalabs/react-ui-kit';
+import { FiArrowUpRight } from 'react-icons/fi';
+
+const FAVORITES: string[] = ['flappy', 'psychedelic'];
```

Comment:
because it's a const, I think you don't need to specify the type

```suggestion
const FAVORITES = ['flappy', 'psychedelic'];
```

Reply:
Done